contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -1066,6 +1066,18 @@ END;
 $BODY$
 LANGUAGE 'plpgsql' STABLE;
 
+CREATE OR REPLACE FUNCTION sys.typeproperty(
+    typename sys.VARCHAR,
+    property sys.VARCHAR
+    )
+RETURNS INT
+AS $$
+DECLARE
+BEGIN
+    RETURN 10;
```

Comment:
Can we return NULL here, instead of hardcoding a value? 
Will that affect SSMS scripting?

Reply:
Done